docs/pages/management/guides/joining-services-kubernetes-serviceaccount.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: Joining Services via Kubernetes ServiceAccount token
+description: Use Kubernetes ServiceAccount tokens to join services running in the same Kubernetes cluster as the Auth service.
+---
+
+This guide will explain how to use the **Kubernetes join method** to configure
+Teleport Services to join your Teleport cluster without sharing any
```

Comment:
```suggestion
Teleport services to join your Teleport cluster without sharing any
```
I know this was capitalized in the Core Concepts page at one point, but I've changed this so it doesn't seem like "Teleport Service" is a discrete product/architectural component